src/CitizenProject/CitizenProjectManagementAuthority.php

```diff
@@ -39,4 +40,18 @@ public function preApprove(CitizenProject $project): void
     {
         $this->manager->preApproveCitizenProject($project);
     }
+
+    public function followCitizenProject(Adherent $adherent, CitizenProject $citizenProject): void
+    {
+        $this->manager->followCitizenProject($adherent, $citizenProject);
+
+        if (!$hosts = $this->manager->getCitizenProjectAdministrators($citizenProject)->toArray()) {
```

Comment:
I think this test should be moved in the listener(s) that don't want to do any kind of processing in this case. Because we will create new listeners in the future and perhaps the gonna want the event even if the CP is orphan.

Reply:
✔️